src/python/grpcio/grpc/_adapter/_c/types/channel.c

```diff
@@ -164,7 +164,7 @@ PyObject *pygrpc_Channel_watch_connectivity_state(
   int last_observed_state;
   CompletionQueue *completion_queue;
   char *keywords[] = {"last_observed_state", "deadline",
-                      "completion_queue", "tag"};
+                      "completion_queue", "tag", NULL};
```

Comment:
... I am at a complete loss. AFAIK that's undocumented in the C API. I guess I'm NULL-terminating everything from now on.